.changeset/config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.1.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "linked": [],
```

Comment:
for lion we probably need linked... I'm for example not 100% sure if I need to release overlays if there is an update to localize 🤔

Reply:
What does lerna do currently? I think it does release all right if you release localize..

Reply:
https://github.com/atlassian/changesets/issues/415 created an issue here to get some clarifications and see if it makes sense for us to use linked packages.

Reply:
Without the usage of `linked` you get a reasonable flow. Given your example - for any release type of `localize` there would always be an accompanying patch bump of overlays. Unless you mark overlays for a minor or major release, of course.

Reply:
now that we added the changeset action it should be pretty clear also what will happen in terms of bumping in the PR that it auto-generates :) so we can see if things make sense before we actually publish. Thanks for taking a look @Andarist

Reply:
No problem. That's a pretty good trait of Changesets - that you can review the release before you actually make it. If you encounter any problems with the tool - just ping me and I will try to help.